docfiles/head.html

```diff
@@ -1,8 +1,8 @@
 <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
 <meta name="viewport" content="width=device-width,height=device-height,user-scalable=no,initial-scale=1.0,maximum-scale=1.0,minimum-scale=1.0">
 
-<link rel="stylesheet" href="/doccdn/semantic.css" />
-<link rel="stylesheet" href="/doccdn/blockly.css" />
+<link rel="stylesheet" href="/blb/semantic.css" />
+<link rel="stylesheet" href="/blb/blockly.css" />
```

Comment:
`doccdn` is replaced on the server - see https://github.com/Microsoft/pxt-backend/blob/master/src/routing.ts#L181

The `docfiles/*` are not uploaded with release - they are taken by the server straight from the `pxt-microbit` repo (and possibly from the appropriate tag of `pxt` repo).

You could check in `doccdn/blockly.css` or whatever in `pxt-microbit` and update it manually when needed - the server will take the `HEAD` version. Or I could look in the `HEAD` of the `-built` repo, but this will have versioning issues.